src/components/text-input/index.md.njk

```diff
@@ -104,6 +104,8 @@ If you need a specific type of information, say so in the input label or hint te
 
 Some users may miss the fact that the input already has a suffix or prefix, and enter a prefix or suffix into the input. Allow for this in your validation and do not show an error.
 
+Prefixes and suffixes are positioned outside of their input so that they don't interfere with the icons that some browsers insert into inputs (for example to show or generate a password).
```

Comment:
```suggestion
Position prefixes and suffixes so that they're outside of their input. This is to avoid interfering with some browsers that might insert an icon (for example to show or generate a password) or autofill some information into the inputs.

Some users may miss that the input already has a suffix or prefix, and enter a prefix or suffix into the input. Allow for this in your validation and do not show an error.
```